thresher/models.py

```diff
@@ -65,11 +65,20 @@ class Topic(models.Model):
     # an id within the given Analysis Type
     topic_id = models.IntegerField() 
 
+    # an id of its parent topic
+    parent_id = models.ForeignKey(self, related_name="subtopics")
+
     # The analysis type to which this topic belongs
-    analysis_type = models.ForeignKey(AnalysisType, related_name='topics')
+    # analysis_type = models.ForeignKey(AnalysisType, related_name='topics')
```

Comment:
Remove rather than comment out.  We have revision control to look back into the past, if we need to!